performance/core/exec/execPromise.js

```diff
@@ -0,0 +1,17 @@
+const exec = require('child_process').exec;
+
+function execPromise(command) {
+    return new Promise(function(resolve, reject) {
+        exec(command, (error, stdout) => {
+            if (error) {
+                console.error(error);
```

Comment:
I know that you hate me... can we remove the console.error?
I have the same identical function in my branch but I don't have the console.error because I don't need it everytime, I actually do something different with the error.

If we have the same function we can eliminate one.

Reply:
Of course I can, I think it was there for no good reason. Ideally we will do something catching the promise no assuming that we log with console :)